src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -337,10 +335,27 @@ const RenderDashboardTags: Renderer<CellProps<DashboardInterface>> = ({ row }) =
   return TagsRenderer(data)
 }
 
+const getBreadcrumbLinks = (folderDetail: { resource: string }, accountId: string, folderString: string) => {
+  if (folderDetail?.resource) {
+    return [
+      {
+        url: routes.toCustomFolderHome({ accountId }),
+        label: folderString
+      },
+      {
+        url: routes.toCustomDashboardHome({ accountId }),
+        label: folderDetail?.resource
```

Comment:
The `if` above asserts that `resource` exists, so you don't need the optional chaining here
```suggestion
        label: folderDetail.resource
```